### DISCUSSION TITLE
is gogs using https://github.com/go-git/ or original git?

### DISCUSSION THREAD

**Discussion by sprappcom** [gogs/gogs#7762] [Q&A]
is gogs using https://github.com/go-git/ or original git?

**Comment by jonnystorm** [comment]
You can see from the project's [`go.mod`](https://github.com/gogs/gogs/blob/f4a78e5cfbd6f4a16b1f33784595d1b3ce3438e9/go.mod#L20) that `github.com/gogs/git-module` is used. This module is a wrapper for the git program, implemented using `os/exec`: https://github.com/gogs/git-module/blob/2c8e00141b0e5674f493a90c3c34a04d58f681bf/command.go#L209
